server/routerlicious/packages/lambdas/src/scribe/lambdaFactory.ts

```diff
@@ -189,22 +189,39 @@ export class ScribeLambdaFactory
 			throw error;
 		}
 
-		if (document.scribe === undefined || document.scribe === null) {
+		const useDefaultCheckpointForNewDocument =
+			document.scribe === undefined || document.scribe === null;
+		const checkpointIsBlank = document.scribe === "";
+		const checkpointQuorumIsScrubbed = isCheckpointQuorumScrubbed(document.scribe);
+		const useLatestSummaryCheckpointForExistingDocument =
+			checkpointIsBlank || !checkpointQuorumIsScrubbed;
+
+		if (useDefaultCheckpointForNewDocument) {
 			// Restore scribe state if not present in the cache. Mongodb casts undefined as null so we are checking
 			// both to be safe. Empty sring denotes a cache that was cleared due to a service summary
 			const message = "New document. Setting empty scribe checkpoint";
 			context.log?.info(message, { messageMetaData });
 			Lumberjack.info(message, lumberProperties);
 			lastCheckpoint = DefaultScribe;
-		} else if (document.scribe === "") {
-			const message = "Existing document. Fetching checkpoint from summary";
+		} else if (useLatestSummaryCheckpointForExistingDocument) {
```

Comment:
Thanks for putting together this PR quickly @znewton! I chatted with Kara about this logic here and realized that we might need to modify it a bit. Since we set useLatestSummaryCheckpointForExistingDocument to true if group DB checkpoint is blank or has quorum data scrubbed and the quorum data in group DB checkpoint is always scrubbed, it means we'll always load checkpoint from summary, which would not be ideal for in-session restarts. We'll need to either differentiate whether we're starting a new session or restarting within a session or check the quorum data after loading checkpoint from DBs (local or group DB). We can discuss more tomorrow.

Reply:
You're right, there was logic that I removed for a comment from @pradeepvairamani and also renamed the function per another comment from Pradeep. I didn't update the logic here.

Reply:
Updated the logic to match the names.